src/LiveComponent/README.md

```diff
@@ -73,7 +73,7 @@ composer require symfony/ux-live-component
 
 This comes with an embedded JavaScript Stimulus controller. Unlike
 other Symfony UX packages, this needs to be enabled manually
-in your `config/bootstrap.js` file:
+in your `assets/bootstrap.js` file:
 
 ```js
 // config/bootstrap.js
```

Comment:
This path should be updated as well I guess?

Reply:
Updated, thank you :)